src/edgeVariablesContainer.ts

```diff
@@ -44,18 +42,24 @@ export class MSPropertyContainer extends variables.BaseVariableContainer {
     }
 
     public async setValue(adapter: EdgeDebugAdapter, name: string, value: string): Promise<string> {
-        const msDebuggerPropertyId = this._childPropertiesMapping[name];
+        const msDebuggerPropertyId = this._childPropertiesMapping.get(name);
 
         if (!msDebuggerPropertyId) {
-            logger.error(`Cannot find msDebuggerPropertyId for {name}`);
-            throw coreUtils.errP(localize("edge.debug.error.notFoundMsDebuggerPropertyId", "Cannot find msDebuggerPropertyId for a property."));
+            // If msDebuggerPropertyId is not present, default to super setValue for this variable.
```

Comment:
Can we use the protocol.json or something similar to figure out what is the contract with Edge, and then validate that contract?
My concern is that if we do, if msDebuggerPropertyId  is available use it, if not don't use it, then we don't have any way of knowing that msDebuggerPropertyId was supposed to be available for this scenario, yet it wasn't.